package.json

```diff
@@ -74,5 +74,13 @@
         "0.1.0": "provideOutlines"
       }
     }
+  },
+  "configSchema": {
+    "additionalGrammars": {
+      "type": "array",
+      "title": "Register Additional Grammars",
+      "default": [],
+      "description": "Register additional html grammars such as text.html.handlebars or text.html.mustache. \n Please note each must be in an array format. []"
```

Comment:
> ... Please note each must be in an array format.

after try this feature locally, I found effective config input format would be something like

`text.html.mustache, foo.bar`

which is bracket-free and quote-free.
Do you think it would be clearer to say something like

*Please note each must be separated by comma.* ?

Reply:
Yes

Reply:
Please force push this PR branch after you edit, thank you.